src/common/hashes/index.ts

```diff
@@ -122,17 +122,17 @@ export const computeSignerListLedgerObjectID = (address: string): string => {
 /**
  * [Offer ID Format](https://xrpl.org/offer.html#offer-id-format)
  *
- * The ID of a Offer object is the SHA-512Half of the following values, concatenated in order:
+ * The index of a Offer object is the SHA-512Half of the following values, concatenated in order:
  *   * The Offer space key (0x006F)
  *   * The AccountID of the account placing the offer
  *   * The Sequence number of the OfferCreate transaction that created the offer
  *
- * This method computes an Offer ID (aka Order ID).
+ * This method computes an Offer Index (aka Order Index).
  *
  * @param address The classic account address of the SignerList owner (starting with r)
- * @returns {string} The ID of the account's Offer object
+ * @returns {string} The index of the account's Offer object
  */
-export const computeOrderID = (address: string, sequence: number): string => {
+export const computeOrderIndex = (address: string, sequence: number): string => {
```

Comment:
```suggestion
export const computeOfferIndex = (address: string, sequence: number): string => {
```
ripple-lib renamed offers to orders, but that's just confusing, so we should undo that